caluma/caluma_form/tests/test_migration.py

```diff
@@ -273,3 +275,95 @@ def test_migrate_to_family_as_pk(transactional_db, caplog):
     assert new_unrelated_doc.family == new_unrelated_doc
     assert new_root_doc.family == new_root_doc
     assert expected_msg in caplog.messages
+
+
+def _col_type_from_db(field):
```

Comment:
This could be imported from `caluma.utils`. Only difference is `_field_name()` is inside `_col_type_from_db` here, but outside in `utils`.

Reply:
Hmm this is a leftover from me refactoring it.. fixed!